scripts/qa_crowbarsetup.sh

```diff
@@ -69,6 +69,7 @@ crowbar_install_log=/var/log/crowbar/install.log
 crowbar_init_api=http://localhost:4567/api
 crowbar_lib_dir=/var/lib/crowbar
 crowbar_api_v2_header="Accept: application/vnd.crowbar.v2.0+json"
+upgrade_progress_file=/var/lib/crowbar/upgrade/6-to-7-progress.yml
```

Comment:
i thought around the definition, something like:

```bash
if [[ "$cloudsource" == "develcloud6" ]] && [[ "$upgrade_cloudsource" == "develcloud7" ]]
    upgrade_progress_file=/var/lib/crowbar/upgrade/6-to-7-progress.yml
fi
```

... btw this code is not tested